tests/e2e/pageobjects/ide/OpenDialogWidget.ts

```diff
@@ -35,7 +35,7 @@ export class OpenDialogWidget {
     constructor(
         @inject(CLASSES.DialogWindow) private readonly dialogWindow: DialogWindow,
         @inject(CLASSES.OpenWorkspaceWidget) private readonly openWorkspaceWidget: OpenWorkspaceWidget,
-        @inject(CLASSES.DriverHelper) private readonly driverHelper: DriverHelper
+        @inject(CLASSES.DriverHelper) private readonly driverHelper: DriverHelper,
```

Comment:
@makambalaji this doesn't look right. There are no more variables after this one, the `,` shouldn't be there.